backend/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM php:7-fpm
+
+RUN pecl install xdebug \
+    && docker-php-ext-enable xdebug \
+    && echo "xdebug.mode=debug" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
+    && echo "xdebug.client_host = host.docker.internal" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
```

Comment:
Since you have docker-compose-dev.yml let's add another Dockerfile for dev, so the prod one is lighter and faster.

Reply:
Never mind. I see the prod docker-compose doesn't build at all and just uses an image.